### PR TITLE
Disable APF [contingency]

### PR DESCRIPTION
**Pull request by lavalamp**
In case https://github.com/kubernetes/kubernetes/pull/96527 causes problems, this is a simple change that disables it.

(only the last commit is new)

```release-note
NONE
```

### CODE REVIEW

**Comment by lavalamp** [comment]
(we can also make a revert PR but this one is very targeted and shouldn't have any merge conflicts)

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -807,7 +807,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	genericfeatures.APIListChunking:         {Default: true, PreRelease: featuregate.Beta},
 	genericfeatures.DryRun:                  {Default: true, PreRelease: featuregate.GA},
 	genericfeatures.ServerSideApply:         {Default: true, PreRelease: featuregate.Beta},
-	genericfeatures.APIPriorityAndFairness:  {Default: true, PreRelease: featuregate.Beta},
+	genericfeatures.APIPriorityAndFairness:  {Default: false, PreRelease: featuregate.Alpha},
```
Comment (adtac) [inline pkg/features/kube_features.go]: staging/src/k8s.io/apiserver/pkg/features/kube_features.go too

Comment (lavalamp) [reply pkg/features/kube_features.go]: thanks, added

**Comment by lavalamp** [comment]
/hold

Prefer https://github.com/kubernetes/kubernetes/pull/96578

(problem with this PR: since the APF tests use the beta types, turning off the feature by default means those tests need to turn on the beta types. Can be fixed but we're super short on time.)

**Comment by lavalamp** [comment]
I looked at the test grid charts and I don't think we'll need this.

**Pull request closed without merging**